test/common-algorithms/sorting/sequential/sequential_mergesort.chpl

```diff
@@ -0,0 +1,100 @@
+
+//merges two subarrays
+//First subarray is arr[l..m]
+//Second subarray is arr[m+1..r]
+
+proc merge(arr, left:int, mid:int, right:int){
+	var i: int = 0, j: int = 0, k: int = 0;
```

Comment:
Be sure to make the same changes to this file as you do for the parallel one, and similarly for the quicksort pair